rust_dev_preview/lambda/src/bin/scenario.rs

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/*
+## Service actions
+
+Service Actions wrap the SDK call, taking a client and any specific parameters necessary for the call.
+
+* CreateFunction
+* GetFunction
+* ListFunctions
+* Invoke
+* UpdateFunctionCode
+* UpdateFunctionConfiguration
+* DeleteFunction
+
+## Scenario
+A scenario runs at a command prompt and prints output to the user on the result of each service action. A scenario can run in one of two ways: straight through, printing out progress as it goes, or as an interactive question/answer script.
+
+## Getting started with functions
+
+Use an SDK to manage Lambda functions: create a function, invoke it, update its code, invoke it again, view its output and logs, and delete it.
+
+This scenario uses two lambda handlers:
+_Note: Handlers don't use AWS SDK API calls_
+
+The increment handler is very simple:
+
+1. It accepts a number, increments it, and returns the new value.
+2. It performs simple logging of the result.
+
+The arithmetic handler is more complex:
+1. It accepts a set of actions ['plus', 'minus', 'times', 'divided-by'] and two numbers, and returns the result of the calculation.
+2. It uses an environment variable to control log level (such as DEBUG, INFO, WARNING, ERROR).
+It logs a few things at different levels, such as:
+    * DEBUG: full event data
+    * INFO: the result of the calculation
+    * WARN~ING~: when a divide by zero error occurs
+    * This will be the typical `RUST_LOG` variable.
```

Comment:
```suggestion
    * DEBUG: Full event data.
    * INFO: The calculation result.
    * WARN~ING~: When a divide by zero error occurs.
    * This will be the typical `RUST_LOG` variable.
```